Journaley/Forms/MainForm.cs

```diff
@@ -475,7 +475,8 @@ private Markdown Markdown
                 if (this.markdown == null)
                 {
                     this.markdown = new Markdown();
-                    ((MarkdownOptions)this.markdown.Options).AutoNewLines = true;
+                    this.markdown.ExtraMode = true;
+                    this.markdown.SafeMode = false;

```

Comment:
Thanks. We might want to turn on the SafeMode option later for resolving #118. Let's keep this false for now.